data/campaigns/Heir_To_The_Throne/scenarios/19c_Cliffs_of_Thoria.cfg

```diff
@@ -188,6 +189,7 @@
         side=4
         no_leader=yes
         controller=ai
+        hidden=yes
```

Comment:
It's the sergeant and he'll join forces with Konrad when you move a unit next to him. I thought about it but there seemed little point for one unit which just stands (lies) there until you do something and, then, the side is empty and needs re-hiding.